environment.rb

```diff
@@ -0,0 +1,139 @@
+module Environment
+    class Depot
+        attr_accessor :packs
+        def initialize 
+            @packs = {
+                :simple_transportation_pack => {
+                    :intelligence => "cellphone",
+                    :items => "mediapack, chevy_versa"
+                },
+                :standard_transportation_pack =>{
+                    :intelligence => "cellphone, antena",
+                    :arsenal => "colt_1911",
+                    :items => "handcuff, mediapack, chemistry, financial"
+                },
+                :simple_mission_pack =>{
+                    :intelligence => "infopack, laptop, cellphone, antenna",
+                    :arsenal => "colt_191",
+                    :items => "handcuffs, mediapack, chemistry, financial"
+                },
+                :standard_mission_pack =>{
+                    :intelligence => "infopack, laptop, cellphone, antenna",
+                    :arsenal => "remington_870, colt_1911, machete, hatchet",
+                    :items => "handcuffs, mediapack, chemistry, financial"
+                }
+            }
+        end
+        
+    end
+
+    class Control
+        attr_accessor :missions
+        @@mission_state = [:paused, :aborted, :failed, :accomplished]
+        def initialize
+            @missions = Hash.new
+        end
+    
+        def new_mission(name:, objective:, pack:)
+            @missions[name]= {
+                    :objective => objective,
+                    :pack => pack,
+                    :status => :active
+                }
+        end
+        @@mission_state.each do |state|
+            define_method("set_mission_to_#{state}") do |arg|
+                @missions[arg][:status] = state
+            end
+        end
+
+    end
+
+    class Human
+        attr_accessor :id, :name, :personal_data, :professional_data, :human
+        
+        def initialize(name:)
+            @id = object_id
+            @name = name
+        end
+        
+        define_method("set_personal_data") do |surname:, age:, country:, language:, marital_status:, children:|
+            @personal_data = {
+                :surname => surname,
+                :age => age,
+                :country => country,
+                :language => language,
+                :marital_status => marital_status,
+                :children => children
+            }
```

Comment:
I think a better way to write this would be something like: 
```
@personal_data = { surname: surname, age: age, country: country, 
                   language: language, marital_status: marital_status, children: children }
```